node/service/src/chain_spec/mod.rs

```diff
@@ -64,6 +71,21 @@ const TEAM_MEMBERS: &[&str] = &[
 	"0x88d388115bd0df43e805b029207cfa4925cecfb29026e345979d9b0004466c49",
 ];
 
+// /// Node `ChainSpec` extensions.
+// ///
+// /// Additional parameters for some Substrate core modules,
+// /// customizable from the chain spec.
+// #[derive(Default, Clone, Serialize, Deserialize, ChainSpecExtension)]
+// #[serde(rename_all = "camelCase")]
+// pub struct Extensions {
+// 	/// Block numbers with known hashes.
+// 	pub fork_blocks: ForkBlocks<OpaqueBlock>,
+// 	/// Known bad block hashes.
+// 	pub bad_blocks: BadBlocks<OpaqueBlock>,
+// 	/// The light sync state extension used by the sync-state rpc.
+// 	pub light_sync_state: LightSyncStateExtension,
+// }
```

Comment:
useless comements?

Reply:
Further review, what's the effect of disabling this?

Reply:
Prepare for the next version.